manifests/mock/puppetlabs_mocks.pp

```diff
@@ -1,5 +1,5 @@
 class rpmbuilder::mock::puppetlabs_mocks (
-  $fedora_releases  = ["14","15","16","17","18","19","20","21"],
+  $fedora_releases  = ["14","15","16","17","18","19","20","21","22"],
```

Comment:
Should I remove all but the currently supported mocks? Are we using fedora 14 for any of the network devices?